components/compose/browser-toolbar/src/main/java/mozilla/components/compose/browser/toolbar/BrowserToolbar.kt

```diff
@@ -36,10 +36,10 @@ import mozilla.components.browser.state.store.BrowserStore
 fun BrowserToolbar(
     store: BrowserStore,
     target: Target,
-    onDisplayMenuClicked: () -> Unit,
```

Comment:
This was on purpose "display" menu since we may also need the option to show a (different) menu in "edit" mode.